src/WingetCreateCLI/Commands/UpdateCommand.cs

```diff
@@ -313,6 +313,8 @@ public Manifests DeserializeManifestContentAndApplyInitialUpdate(List<string> la
                 manifests = ConvertSingletonToMultifileManifest(manifests.SingletonManifest);
             }
 
+            UpdateManifestVersionToLatest(manifests);

```

Comment:
>UpdateManifestVersionToLatest [](http://example.com/codeflow?start=12&length=29)

Curious why is update always updating to latest schema version? Is there an option to a specific schema version?

Reply:
We want to keep manifests as current as reasonable. We've been discussing enabling an experimental feature to try newer versions of the schema early. I'm not sure why we would want to support an older schema other than the scenario where a user is running an older version of a REST source.

Reply:
Keeping schema version as latest as possible is probably ok for v1.0 and v1.1 schemas because they are compatible. But if we have a breaking schema version like v2.0(given we still do not drop v1.0 schemas), this will be an issue. For example if a user calls "winget-create update <SomeV1.0Manifest>" to maybe just update some description fields or installer url of a manifest, suddenly the manifest becomes a v2.0 manifest and the package lost support from old clients where only v1.0 is recognized. We may certainly encourage latest schema version, but I think we should still support schema versions  that are not deprecated yet.

Reply:
I have created a new feature issue to track this here: #225